src/Arcus.Observability.Tests.Runtimes.AzureFunction/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR "/src/Arcus.Observability.Tests.Runtimes.AzureFunction"
 RUN dotnet build "Arcus.Observability.Tests.Runtimes.AzureFunction.csproj" -c Release -o /app/build
 
 FROM build AS publish
-RUN dotnet publish "Arcus.Observability.Tests.Runtimes.AzureFunction.csproj" -c Release -o /app/publish
+RUN dotnet publish "Arcus.Observability.Tests.Runtimes.AzureFunction.csproj" -c Release -o /app/publish /p:UseAppHost=false
```

Comment:
I'm wondering why we need this.

I've been reading a bit the docs on this option, but I'm afraid I don't really understand :)

I understand that by using this option, we're falling back on the 'framework dependent deployment' whereas since .NET 3, 'framework dependent executable' is the default mode.
With FDD, a dll is generated and that dll is started, whereas with FDE, an exe is created.   Why do we need to fallback to something that is not the default mode anymore ?

Reply:
Honestly, this is close to Chinese to me most of the time, no matter how much a spend time in the Docker-Wonderworld. To fix the Docker issues, I started small again (minimal), with the default contents that are in the Microsoft project templates. This option was there, so I left it, but if you say this is not what we want; we can off course remove it.

Reply:
I wonder if it breaks if you remove it :)

Reply:
Let's try.

Reply:
It works 👍🎉